priv/repo/migrations/20170809102200_emails_location_for_prod.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Askbywho.Repo.Migrations.EmailsLocationForProd do
+  use Ecto.Migration
+
+  def change do
+    alter table(:emails) do
+      add :latitude, :real
+      add :longitude, :real
+      add :location, :string
```

Comment:
was thinking if it is dynamic, does it make sense to save? so the city is nice to see, but saving lat and long as it is more versatile